### PR TITLE
Complete overhaul of cookbook

### PR DESCRIPTION
**Pull request by Sauraus**
- Install SonarQube 5.0.1
- Update sonar.properties for 5.x
- Added Percona DB support
- rename local_database to postgresql recipe
- Migrated to Berks 3.x
- Install java runtime as part of Sonar server install

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name: chef_solo
+
+platforms:
+  - name: ubuntu-12.04
+  - name: centos-6.4
+
+suites:
+  - name: default
+    run_list:
+      - recipe[sonar::default]
```
Comment (yurinnick) [inline .kitchen.yml]: Please fix recipe list to get sonar running, sonar::default recipe is now empty

Comment (Sauraus) [inline .kitchen.yml]: Sorry about that I use Berkshelf mostly and neglected the Kitchen.

I am off to Chef Conf 2015 later today so updates may be a few days late. ;)

**Comment by Sauraus** [comment]
I am also adding the option to configure the location of data/temp&logs directories to the cookbook right now.

And I am moving the server recipe to use the 'ark' cookbook instead of unzip etc.

File: .kitchen.yml (hunk 1 above)
Comment (yurinnick) [inline .kitchen.yml]: Also I could recommend you to try to use kitchen-docker, which could really be much faster then vagrant, but I am not really sure how stable it is.

**Comment by yurinnick** [comment]
It's really cool! Looking forward for your updates.

**Comment by Sauraus** [comment]
I am currently at Chef Conf and can't run the entire Kitchen suite, but single VMs are coming up just fine using vagrant & berkshelf.

**Pull request closed without merging**